doc/aliases/add_toggle.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/javascript">
+/*
+  Creates a toggleable div that includes
+  the text/code associated to a button.
+*/
+var $getDiv = $('div.newInnerHTML').last();
+var buttonName = $getDiv.html();
+var label = getLabelName(buttonName.trim());
```

Comment:
I don't think we need to process button names every time they are added, because they are constant.